Sprint-2/stretch/count-words.js

```diff
@@ -26,3 +26,55 @@
 
 3. Order the results to find out which word is the most common in the input
 */
+function countWords(inputString) {
+  // Step 1: Initialize an empty object to store the word counts
+  const wordCount = {};// This object will store the word counts. The word itself will be the key, and the count (number of times the word appears) will be the value
+
+  // Step 2: Convert the string to lowercase and remove punctuation
+  // Use regular expression to remove punctuation
+  inputString = inputString.toLowerCase().replace(/[.,!?]/g, ""); //We use toLowerCase() to make sure that we don't differentiate between upper and lower case letters (e.g., "Hello" and "hello" should be counted as the same word), replace(/[.,!?]/g, "") uses a regular expression to remove common punctuation marks (like ., ,, !, ?) from the string to tidy it up.
+
+  // Step 3: Split the string into individual words
+  const words = inputString.split(" "); //The split(" ") method splits the string into an array of words by using the space character as a delimiter.
+
+  // Step 4: Loop through each word
+  for (let word of words) {
+    // If the word already exists in the wordCount object, increment its value
+    if (wordCount[word]) {
+      wordCount[word]++;
+    } else {
+      // Otherwise, set the word's count to 1
+      wordCount[word] = 1;
+    }
+  }
+
+  // Step 5: Return the wordCount object
+  return wordCount;
+}
+console.log(countWords("charger and computer and terminal"));
```

Comment:
What do you think `countWords("hello.world       hello world hello!!!!world")` should return?
Does your function returns what you expect?